linera-service/tests/end_to_end_tests.rs

```diff
@@ -279,17 +280,17 @@ impl AmmApp {
 #[cfg_attr(feature = "kubernetes", test_case(SharedLocalKubernetesNetTestingConfig::new(Network::Grpc, BuildArg::Build) ; "kubernetes_grpc"))]
 #[cfg_attr(feature = "remote_net", test_case(RemoteNetTestingConfig::new(None) ; "remote_net_grpc"))]
 #[test_log::test(tokio::test)]
-async fn test_wasm_end_to_end_counter(config: impl LineraNetConfig) {
+async fn test_wasm_end_to_end_counter(config: impl LineraNetConfig) -> Result<()> {
     use counter::CounterAbi;
     let _guard = INTEGRATION_TEST_GUARD.lock().await;
 
-    let (mut net, client) = config.instantiate().await.unwrap();
+    let (mut net, client) = config.instantiate().await?;
 
     let original_counter_value = 35;
     let increment = 5;
 
-    let chain = client.get_wallet().unwrap().default_chain().unwrap();
-    let (contract, service) = client.build_example("counter").await.unwrap();
+    let chain = client.get_wallet()?.default_chain().unwrap();
```

Comment:
Or even `client.get_wallet()?.default_chain()?`? (Also in the other tests.)

Reply:
Unfortunately, that's not possible because `default_chain()` returns an `Option<_>`, and we can't convert automatically the `None` value into `anyhow::Error` ([yet?](https://github.com/rust-lang/rust/issues/84277)).